bin/start-postgres-localstack

```diff
@@ -108,6 +108,9 @@ kubectl -n harmony create configmap localstack-config --from-literal=startup.py=
 
 kubectl apply -n harmony -f ./config/local-postgres-localstack-deployment.yml
 
+# run the database migrations for postgres
+NODE_ENV=production DATABASE_URL=postgresql://postgres:password@localhost:5432  npx knex --cwd db migrate:latest
```

Comment:
No, I still have to change HIAB to use postgres instead of sqlite, but I just have to do that before we make queues the default.

Reply:
Please sanity check HIAB with the default queues disabled before merging this in.

Reply:
I removed this line for now since there is a race condition waiting for postgres to become ready to accept SQL commands. This line was just intended as a convenience and has nothing to do with queueing.